canvas_modules/harness/src/client/components/sidepanel-canvas.jsx

```diff
@@ -835,270 +844,299 @@ export default class SidePanelForms extends React.Component {
+						value={TOOLBAR_TYPE_DEFAULT}
+						labelText={TOOLBAR_TYPE_DEFAULT}
+					/>
+					<RadioButton
+						value={TOOLBAR_TYPE_SINGLE_BAR}
+						labelText={TOOLBAR_TYPE_SINGLE_BAR}
+					/>
+					<RadioButton
+						value={TOOLBAR_TYPE_BEFORE_AFTER}
+						labelText={TOOLBAR_TYPE_BEFORE_AFTER}
+					/>
+					<RadioButton
+						value={TOOLBAR_TYPE_CUSTOM_RIGHT_SIDE}
+						labelText={TOOLBAR_TYPE_CUSTOM_RIGHT_SIDE}
+					/>
+					<RadioButton
+						value={TOOLBAR_TYPE_CARBON_BUTTONS}
+						labelText={TOOLBAR_TYPE_CARBON_BUTTONS}
+					/>
+					<RadioButton
+						value={TOOLBAR_TYPE_CUSTOM_ACTIONS}
+						labelText={TOOLBAR_TYPE_CUSTOM_ACTIONS}
+					/>
+				</RadioButtonGroup>
+			</FormGroup>
+		</div>);
+
+		var tipConfig = (<div className="harness-sidepanel-children" id="harness-sidepanel-tip-config">
+			<fieldset className="bx--fieldset">
+				<legend className="bx--label">Tips</legend>
+				<Checkbox
+					id="tip_palette"
+					labelText={TIP_PALETTE}
+					onChange={this.tipConfigChange}
+					checked={this.props.getStateValue("selectedTipConfig").palette}
 				/>
-				<RadioButton
-					value={TOOLBAR_TYPE_CUSTOM_RIGHT_SIDE}
-					labelText={TOOLBAR_TYPE_CUSTOM_RIGHT_SIDE}
+				<Checkbox
+					id="tip_nodes"
+					labelText={TIP_NODES}
+					onChange={this.tipConfigChange}
+					checked={this.props.getStateValue("selectedTipConfig").nodes}
 				/>
-				<RadioButton
-					value={TOOLBAR_TYPE_CARBON_BUTTONS}
-					labelText={TOOLBAR_TYPE_CARBON_BUTTONS}
+				<Checkbox
+					id="tip_ports"
+					labelText={TIP_PORTS}
+					onChange={this.tipConfigChange}
+					checked={this.props.getStateValue("selectedTipConfig").ports}
 				/>
-				<RadioButton
-					value={TOOLBAR_TYPE_CUSTOM_ACTIONS}
-					labelText={TOOLBAR_TYPE_CUSTOM_ACTIONS}
+				<Checkbox
+					id="tip_links"
+					labelText={TIP_LINKS}
+					onChange={this.tipConfigChange}
+					checked={this.props.getStateValue("selectedTipConfig").links}
 				/>
-			</RadioButtonGroup>
-		</div>);
-
-		var tipConfig = (<div className="harness-sidepanel-children" id="harness-sidepanel-tip-config">
-			<div className="harness-sidepanel-headers">Tips</div>
-			<Checkbox
-				id="tip_palette"
-				labelText={TIP_PALETTE}
-				onChange={this.tipConfigChange}
-				checked={this.props.getStateValue("selectedTipConfig").palette}
-			/>
-			<Checkbox
-				id="tip_nodes"
-				labelText={TIP_NODES}
-				onChange={this.tipConfigChange}
-				checked={this.props.getStateValue("selectedTipConfig").nodes}
-			/>
-			<Checkbox
-				id="tip_ports"
-				labelText={TIP_PORTS}
-				onChange={this.tipConfigChange}
-				checked={this.props.getStateValue("selectedTipConfig").ports}
-			/>
-			<Checkbox
-				id="tip_links"
-				labelText={TIP_LINKS}
-				onChange={this.tipConfigChange}
-				checked={this.props.getStateValue("selectedTipConfig").links}
-			/>
+			</fieldset>
 		</div>);
 
 		var extraCanvas = (<div className="harness-sidepanel-children" id="harness-sidepanel-extra-canvas">
 			<form>
```

Comment:
Should this `<form>` element, and the ones below, be changed to `<FormGroup>` for consistency?